stylesheets/all.scss

```diff
@@ -89,10 +89,9 @@ $screen-sm-min: 576px;
 .down-arrow-icon {
   width: 15px;
   height: 15px;
-  top: 3px;
+  top: 7px;
   position: relative;
   cursor: pointer;
-  right: 25px;
 }
```

Comment:
this is only being used in autocomplete which has been updated in my pr to this.

Reply:
this is breaking the autocomplete ui, we should keep it in mind if we want to deploy this before the autocomplete